feature/experimental/p4rt/tests/p4rt_mastership/p4rt_mastership.MD

```diff
@@ -0,0 +1,22 @@
+# P4RT-2.1: P4RT Mastership
+
+## Summary
+
+Validate the P4RT server handles master election and failover
+
+## Procedure
+
+* Enable P4RT on a single FAP by configuring an ID on the device and one or more interfaces.
```

Comment:
Primary/Secondary over master/slave?

Reply:
@Ankur19, Should we rename the test to P4RT election instead of mastership to keep the trend of inclusive language?